packages/devtools-reps/src/object-inspector/components/ObjectInspectorItem.js

```diff
@@ -163,6 +167,18 @@ class ObjectInspectorItem extends Component<Props> {
         };
       }
 
+      if (nodeHasGetter(item)) {
+        const parentNode = getParent(item);
+        const parentGripNode = parentNode && getClosestGripNode(parentNode);
+        const parentGrip = parentGripNode && getValue(parentGripNode);
+        if (parentGrip) {
+          Object.assign(repProps, {
+            onInvokeGetterButtonClick: () =>
```

Comment:
still not sure why it is conditionally availabe. Could we have an onClick callback that does this check and then invokeGetter?

Reply:
because we only want to show the button in expanded nodes.
So given the following object: 

```js
x = {
  get a() {}
}
```

when printed, we display: `▶︎ Object { a : Getter}`. Here we don't want to be able to invoke the getter, because it would add too much complexity for now.

But we do want to show it when expanded, i.e. 
```
▼ Object
| a: Getter >>
| <get a>: function
```

does it make sense ?